documentation/book/assembly-access-configuration-kafka-mirror-maker.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-access-configuration-kafka-mirror-maker-{context}']
+= Configuring Kafka Mirror Maker access control
```

Comment:
At least for me, access control would be authorization. But that is done rather on the broker side. I would probably use here _authentication_ instead of _access control_

Reply:
I chose access control as an all encompassing term, but if you think authentication is more accurate let's go with that.